cub/cub/util_temporary_storage.cuh

```diff
@@ -0,0 +1,93 @@
+/******************************************************************************
+ * Copyright (c) 2011, Duane Merrill.  All rights reserved.
```

Comment:
Does that copyright still apply?

Reply:
Yeah, it seems `AliasTemporaries ` was added by Duane ~10 yrs ago: https://github.com/NVIDIA/cub/commit/71da6570e32c1c967a1feff9f49c7186f7b64de1#diff-c2367b4150bdbd82170e49217a43ddbe1d1f885154f225da30fa305645f7507a